src/python/pants/backend/jvm/tasks/ivy_imports.py

```diff
@@ -28,6 +29,9 @@ def has_imports(target):
         return isinstance(target, ImportJarsMixin) and target.imported_targets
 
     def execute(self):
+        deprecated_conditional(
```

Comment:
You can just use `warn_or_error`.

Reply:
This validates the semver though, which `warn_or_error` doesn't.